runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -296,6 +296,7 @@
 	TOTAL_POINTS(5431),
 	PERSONAL_POINTS(5422),
 	RAID_PARTY_SIZE(5424),
+	RAID_STATE(5425),
```

Comment:
Doc for this var would be helpful, even if it's something as simple as `0 = raid not started, >0 = raid started`